roles/kubernetes/control-plane/defaults/main/main.yml

```diff
@@ -228,3 +228,6 @@ auto_renew_certificates_systemd_calendar: "{{ 'Mon *-*-1,2,3,4,5,6,7 03:' ~
 # If we have requirement like without renewing certs upgrade the cluster,
 # we can opt out from the default behavior by setting kubeadm_upgrade_auto_cert_renewal to false
 kubeadm_upgrade_auto_cert_renewal: true
+
+# Bash alias of kubectl to interact with Kubernetes cluster much easier
+kubectl_alias: k
```

Comment:
Thanks @tu1h 

Using the k alias as the  default behavior seems not common.
Should we do not change the default behavior ? using
```
# kubectl_alias: k
```